_opensearch/rest-api/analyze-apis/perform-text-analysis.md

```diff
@@ -0,0 +1,670 @@
+      "start_offset" : 3,
+      "end_offset" : 23,
+      "type" : "word",
+      "position" : 0
+    }
+  ]
+}
+````
+
+<hr />
+
+You can combine filters using an array.
+
+The following request combines a `lowercase` translation with a `stop` filter that removes the words in the `stopwords` array:
+
+````json
+GET /_analyze
+{
+  "tokenizer" : "whitespace",
+  "filter" : ["lowercase", {"type": "stop", "stopwords": [ "to", "in"]}],
+  "text" : "how to train your dog in five steps"
+}
+````
+
+The previous request returns the following fields:
+
+````json
+{
+  "tokens" : [
+    {
+      "token" : "how",
+      "start_offset" : 0,
+      "end_offset" : 3,
+      "type" : "word",
+      "position" : 0
+    },
+    {
+      "token" : "train",
+      "start_offset" : 7,
+      "end_offset" : 12,
+      "type" : "word",
+      "position" : 2
+    },
+    {
+      "token" : "your",
+      "start_offset" : 13,
+      "end_offset" : 17,
+      "type" : "word",
+      "position" : 3
+    },
+    {
+      "token" : "dog",
+      "start_offset" : 18,
+      "end_offset" : 21,
+      "type" : "word",
+      "position" : 4
+    },
+    {
+      "token" : "five",
+      "start_offset" : 25,
+      "end_offset" : 29,
+      "type" : "word",
+      "position" : 6
+    },
+    {
+      "token" : "steps",
+      "start_offset" : 30,
+      "end_offset" : 35,
+      "type" : "word",
+      "position" : 7
+    }
+  ]
+}
+````
+
+#### Specify an index
+
+You can analyze text using an index's default analyzer or you can specify a different analyzer.
+
+The following request analyzes the provided text using the default analyzer associated with the `books` index:
+
+````json
+GET /books/_analyze
+{
+  "text" : "OpenSearch analyze test"
+}
+````
+
+The previous request returns the following fields:
+
+````json
+
+  "tokens" : [
+    {
+      "token" : "opensearch",
+      "start_offset" : 0,
+      "end_offset" : 10,
+      "type" : "<ALPHANUM>",
+      "position" : 0
+    },
+    {
+      "token" : "analyze",
+      "start_offset" : 11,
+      "end_offset" : 18,
+      "type" : "<ALPHANUM>",
+      "position" : 1
+    },
+    {
+      "token" : "test",
+      "start_offset" : 19,
+      "end_offset" : 23,
+      "type" : "<ALPHANUM>",
+      "position" : 2
+    }
+  ]
+}
+````
+
+<hr />
+
+The following request analyzes the provided text using the `keyword` analyzer, which returns the entire text value as a single token:
+
+````json
+GET /books/_analyze
+{
+  "analyzer" : "keyword",
+  "text" : "OpenSearch analyze test"
+}
+````
+
+The previous request returns the following fields:
+
+````json
+{
+  "tokens" : [
+    {
+      "token" : "OpenSearch analyze test",
+      "start_offset" : 0,
+      "end_offset" : 23,
+      "type" : "word",
+      "position" : 0
+    }
+  ]
+}
+````
+
+#### Derive the analyzer from an index field
+
+You pass text and a field in the index. The API looks up the field's analyzer and uses that analyzer to analyes the text. 
```

Comment:
This tense and voice seems out of the blue. Are you describing the request? Did you mean "You **can** pass text..."? 
Second sentence, To me it's clear just using the pronoun "it" for the antecedent: "The API looks up the field's analyzer and uses it to analyze the text." Then you don't bulk up a relatively short sentence with the same word.
Also, typo on "analyes".